hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/BasicRecordStoreLoader.java

```diff
@@ -74,8 +74,8 @@
     }
 
```

Comment:
Due to this refactoring where the methods were moved to inner class it's super tough to read what has changed. 
Could we revert this refactoring and then have a separate commit where the methods are moved...